compiler/ast_to_ir.py

```diff
@@ -378,7 +389,7 @@ def execute_shell_asts(asts):
     save_asts_json(asts, ir_filename)
     output_script = from_ir_to_shell(ir_filename)
     # log(output_script)
-    exec_obj = subprocess.run(["/usr/bin/env", "bash"], input=output_script,
+    exec_obj = subprocess.run(["/bin/bash"], input=output_script,
```

Comment:
A user had an issue with this. Did you change it back on purpose? If so we should find a solution that works for everyone.